app/uk/gov/hmrc/soletraderidentificationfrontend/connectors/SoleTraderIdentificationConnector.scala

```diff
@@ -16,44 +16,55 @@
 
 package uk.gov.hmrc.soletraderidentificationfrontend.connectors
 
-import play.api.libs.json.{Reads, Writes}
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpReadsInstances}
+import play.api.libs.json.{Json, Reads, Writes}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpReadsInstances, StringContextOps}
 import uk.gov.hmrc.soletraderidentificationfrontend.config.AppConfig
 import uk.gov.hmrc.soletraderidentificationfrontend.httpParsers.RemoveSoleTraderDetailsHttpParser._
 import uk.gov.hmrc.soletraderidentificationfrontend.httpParsers.RetrieveIndividualDetailsHttpParser.RetrieveIndividualDetailsHttpReads
 import uk.gov.hmrc.soletraderidentificationfrontend.httpParsers.SoleTraderIdentificationStorageHttpParser._
 import uk.gov.hmrc.soletraderidentificationfrontend.models.{IndividualDetails, SoleTraderDetails}
+import uk.gov.hmrc.http.client.HttpClientV2
 
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class SoleTraderIdentificationConnector @Inject() (http: HttpClient, appConfig: AppConfig)(implicit ec: ExecutionContext) extends HttpReadsInstances {
+class SoleTraderIdentificationConnector @Inject() (http: HttpClientV2, appConfig: AppConfig)(implicit ec: ExecutionContext)
+    extends HttpReadsInstances {
```

Comment:
Using "import uk.gov.hmrc.http.HttpReads.Implicits._" instead of directly importing HttpReadsInstances would allow you to remove HttpReadsInstances as an extension.